### ISSUE TITLE
Link actions between scatterplot and world map

### ISSUE THREAD

**Issue by lmar76** [ESA-VirES/Aeolus-Client#50]
According to SR-VIR_AE-FUN_00160 bullet 1:

> moving the mouse on the scatterplot the relevant position on the map shall be shown; the other way around shall also be possible (i.e. moving the mouse over the map, the relevant data on the scatterplot shall be shown) 

The second part of the requirement is not implemented

**Comment by santilland** [comment]
Correct, the second part is not implemented, it would be fairly straight forward to do this for point (1D) data, as the picking method is basically supported by the underlying library.
For picking a specific value inside the curtain things are quite more complicated.
